### PR TITLE
Download count to dashboard

### LINKED ISSUE #432: Add download count to public dataset list
Related to #430

### PR DESCRIPTION
**Pull request by olgn**
fixes #432 

* dataset usage statistics are now added to each dataset returned by the dashboard dataset store.
* dataset downloads are added to the metrics component in the dataset page
* dataset downloads are added to the public dataset dashboard
* dataset dashboard is filterable by download count

### CODE REVIEW

File: app/src/scripts/common/partials/metric.jsx [hunk 1]
```diff
@@ -20,7 +20,12 @@ export default class Metric extends React.Component {
       case 'stars':
         spanClass = 'dataset-status ds-primary'
         iconClass = 'fa fa-star'
-        tip = value + ' users like this dataset'
+        tip = 'This dataset has ' + value + ' likes.'
+        break
+      case 'downloads':
+        spanClass = 'dataset-status ds-primary'
+        iconClass = 'fa fa-download'
+        tip = 'This dataset has ' + value + ' download.'
```
Comment (nellh) [inline app/src/scripts/common/partials/metric.jsx]: "downloads"

**Review by nellh — CHANGES_REQUESTED** [review]
This looks like it breaks the follow dataset dialogue and does not actually count downloads.

**Comment by nellh** [comment]
Clarifying the issue, drafts show a download count but downloads for a draft are not actually counted. The follow page does display as it should.

**Comment by olgn** [comment]
@nellh those review points have been addressed

**Review by nellh — APPROVED** [review]

**Pull request merged**